ultradns-rest/src/main/java/denominator/ultradns/UltraDNSRestProvider.java

```diff
@@ -71,16 +86,35 @@ public String url() {
    */
   @Override
   public Map<String, Collection<String>> profileToRecordTypes() {
-    Map<String, Collection<String>>
-            profileToRecordTypes =
-            new LinkedHashMap<String, Collection<String>>();
-    profileToRecordTypes.put("geo",
-            Arrays
-                    .asList("A", "AAAA", "CNAME", "HINFO", "MX", "NAPTR", "PTR", "RP",
-                            "SRV", "TXT"));
-    profileToRecordTypes.put("roundRobin",
-            Arrays.asList("A", "AAAA", "HINFO", "MX", "NAPTR", "NS", "PTR", "RP",
-                    "SPF", "SRV", "TXT"));
+    Map<String, Collection<String>> profileToRecordTypes = new LinkedHashMap<String, Collection<String>>();
+
+    profileToRecordTypes.put("geo", Arrays.asList(
+            ResourceTypes.A.name(),
+            ResourceTypes.AAAA.name(),
+            ResourceTypes.CNAME.name(),
+            ResourceTypes.HINFO.name(),
+            ResourceTypes.MX.name(),
+            ResourceTypes.NAPTR.name(),
+            ResourceTypes.PTR.name(),
+            ResourceTypes.RP.name(),
+            ResourceTypes.SRV.name(),
+            ResourceTypes.TXT.name()
+    ));
+
+    profileToRecordTypes.put("roundRobin", Arrays.asList(
+            ResourceTypes.A.name(),
```

Comment:
I disagree, what if someone, in future, wanted to update the collection & only do it at one place & misses the second one?